### PR TITLE
add Sodium page

### PR DESCRIPTION
**Pull request by davidhuang317**
added information about sodium

### CODE REVIEW

File: elements/sodium.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: page
+title: Sodium
+---
+
```
Comment (franbiancaoca) [inline elements/sodium.md:5]: Code looks like it was done correctly on sodium

**Review by franbiancaoca — COMMENTED** [review]
Great job on completing the first assignment!

**Review by lkollmorgen — APPROVED** [review]
hmm can't find a fun fact 🙄

File: elements/sodium.md (hunk 1 above)
Comment (franbiancaoca) [reply elements/sodium.md:5]: Actually - the page is missing three sentences about sodium!

**Pull request closed without merging**